### PR TITLE
fix(docs): Flux function updates

### PR DESCRIPTION
**Pull request by kelseiv**
Add new Flux functions for Flux 0.36 - 0.39

### CODE REVIEW

File: ui/src/shared/constants/fluxFunctions.ts [hunk 1]
```diff
@@ -977,6 +1142,79 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
...
+        name: 'column',
+        desc: 'The column to operate on. Defaults to `"_value"`.',
+        type: 'String',
+      },
+    ],
+    package: '',
+    desc:
+      'Applies the Holt-Winters forecasting method to input tables. The Holt-Winters method predicts `n` seasonally-adjusted values for the specified `column` at the specified `interval`.',
+    example:
+      'holtWinters(n: 10, seasonality: 4, interval: 30d, withFit: false, column: "_value")',
```
Comment (sanderson) [inline ui/src/shared/constants/fluxFunctions.ts]: Let's use a simpler example here. Not all the parameters are required and users can add them as they need them. Change this example to:

```
holtWinters(n: 10, interval: 1d)
```

File: ui/src/shared/constants/fluxFunctions.ts [hunk 2]
```diff
@@ -977,6 +1142,79 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
...
+      {
+        name: 'timeColumn',
+        desc: 'The column that contains the time value. Default is `"_time"`.',
+        type: 'String`',
+      },
+    ],
+    package: '',
+    desc:
+      'Retains all rows with time values in a specified hour range. Hours are specified in military time.',
+    example: 'hourSelection(start: 9, stop: 17, timeColumn: "_time")',
```
Comment (sanderson) [inline ui/src/shared/constants/fluxFunctions.ts]: Remove the `timeColumn` param from this example to keep it simple.

```
hourSelection(start: 9, stop: 17)
```

File: ui/src/shared/constants/fluxFunctions.ts [hunk 3]
```diff
@@ -250,6 +250,28 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
...
+      {
+        name: 'columns',
+        desc: 'Columns to operate on. Defaults to `["_value"]`.',
+        type: 'Array of Strings`',
+      },
+    ],
+    package: '',
+    desc:
+      'Applies the technical momentum indicator developed by Tushar Chande.',
+    example: 'chandeMomentumOscillator(n: 10, columns: ["_value"])',
```
Comment (sanderson) [inline ui/src/shared/constants/fluxFunctions.ts]: Remove the `columns` param from this example to keep it simple. It's rare that a user will want to using something other than the default value here.

```
chandeMomentumOscillator(n: 10)
```

File: ui/src/shared/constants/fluxFunctions.ts [hunk 4]
```diff
@@ -2744,6 +2982,28 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
...
+      },
+      {
+        name: 'columns',
+        desc: 'Columns to operate on. Defaults to `["_value"]`.',
+        type: 'Array of Strings`',
+      },
+    ],
+    package: '',
+    desc: 'Measures the relative speed and change of values in an input table.',
+    example: 'relativeStrengthIndex(n: 5, columns: ["_value"])',
```
Comment (sanderson) [inline ui/src/shared/constants/fluxFunctions.ts]: Remove the `columns` param from the example:

```
relativeStrengthIndex(n: 5)
```

File: ui/src/shared/constants/fluxFunctions.ts [hunk 5]
```diff
@@ -3760,6 +4030,29 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
...
+        name: 'offset',
+        desc:
+          'The number of records to skip at the end of a table before limiting to `n`. Defaults to `0`.',
+        type: 'Integer',
+      },
+    ],
+    package: '',
+    desc:
+      'Limits each output table to the last `n` records, excluding the offset.',
+    example: 'tail(n: 10, offset: 0)',
```
Comment (sanderson) [inline ui/src/shared/constants/fluxFunctions.ts]: Remove the `offset` param from the example:

```
tail(n: 10)
```

File: ui/src/shared/constants/fluxFunctions.ts [hunk 6]
```diff
@@ -570,14 +716,21 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
...
+        desc:
+          'Indicates the first row should be kept. If `true`, the difference will be `null`. Defaults to `false`.',
+        type: 'Boolean',
       },
     ],
     package: '',
     desc:
-      'Computes the difference between subsequent records in the specified column.',
+      'Computes the difference between subsequent non-null records in the specified columns.',
     example: 'difference(nonNegative: false, column: "_value")',
```
Comment (russorat) [inline ui/src/shared/constants/fluxFunctions.ts]: `difference(nonNegative: false, columns: ["_value"])`

**Pull request merged**